frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -62,6 +63,9 @@ export const getOperatingSystemName = (vmLike: VMGenericLikeEntityKind) =>
     `${TEMPLATE_OS_NAME_ANNOTATION}/${getOperatingSystem(vmLike)}`,
   );
 
+export const getTemplateName = (vm: VMGenericLikeEntityKind) =>
```

Comment:
there is already a function `getVMTemplateNamespacedName` - can we refactor it so it uses these constants?

Reply:
this function is not needed anymore. Can we remove it?